tests/providers/google/cloud/operators/test_cloud_storage_transfer_service.py

```diff
@@ -324,21 +324,33 @@ def test_job_create_multiple(self, aws_hook, gcp_hook):
     # (could be anything else) just to test if the templating works for all
     # fields
     @pytest.mark.db_test
+    @pytest.mark.parametrize(
+        "body, excepted",
+        [
+            (
+                {"description": "{{ dag.dag_id }}"},
+                {
+                    "description": "TestGcpStorageTransferJobCreateOperator_test_templates",
+                },
+            ),
+            ("{{ dag.dag_id }}", "TestGcpStorageTransferJobCreateOperator_test_templates"),
```

Comment:
Is this a valid body?
https://github.com/apache/airflow/blob/d77f115ebc37613a48563a760a2df4e01f604335/airflow/providers/google/cloud/operators/cloud_storage_transfer_service.py#L249-L250

Reply:
it's not, but it just for checking the rendering here.

Reply:
Change to valid body